app/src/main/java/Fragments/FragmentMain.java

```diff
@@ -11,8 +11,23 @@
 
 import com.omniimpact.javaexperiments.R;
 
+/**
+ * This class represents the main Fragment UI element, which
+ * is the main screen that users start on and return to when
+ * using the application.
+ */
 public class FragmentMain extends Fragment {
 
+    /**
+     * Occurs when creating a new FragmentMain object.
+     * In this case, since it is the main Fragment, it loads
+     * the main fragment layout to show the user.
+     *
+     * @param inflater TODO: Yeah let's explain this
+     * @param container TODO: "
```

Comment:
This is the parent container, provided to the layout inflater in order to make sure the view is displayed at the correct size and proportions